### ISSUE TITLE
Parameter with specific name

### ISSUE THREAD

**Issue by vinaybedre** [developit/preact-router#250]
Hi,
This is not working for me. I am unable to navigate to List, always i am being redirected to Details

```
<Router onChange={this.handleRoute}>
					<Home path="/" />
					<Profile path="/profile/" user="me" />
					<Profile path="/profile/:user" />
					<List path="/data/popular-documents" />
					<Details path="/data/:id" />
				</Router>
```

**Comment by developit** [comment]
This should be fixed when we merge #242.

**Comment by vinaybedre** [comment]
after https://github.com/developit/preact-router/pull/242 is merged, issue is solved. 
Thanks
